web/src/features/charts/elements/ValueAxis.tsx

```diff
@@ -12,28 +12,27 @@ function ValueAxis({ scale, width, formatTick }: ValueAxisProps) {
   const [y1, y2] = scale.range();
   return (
     <g
-      transform={`translate(${width - 1} -1)`}
+      transform={`translate(${width} 0)`}
```

Comment:
not sure why there was a (1,1)px offset. It looks better without it

Reply:
I think it's due to AM/PM making it potentially overlap/go out of bounds but I agree.